lumen/ai/coordinator.py

```diff
@@ -448,6 +448,7 @@ async def respond(self, messages: list[Message], **kwargs: dict[str, Any]) -> st
                 self.interface.stream(msg, user='Lumen')
                 return msg
             for node in execution_graph:
+                print(node.agent)
```

Comment:
If this is for debugging purposes, you should remove the print statement. If it is for logging purposes, then you should make a logger.